jsweb/exercises/dom_access.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Exercise: DOM Access</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="stylesheet" type="text/css" href="../../common/bootstrap-responsive.css">
+    <link rel="stylesheet" type="text/css" href="../../common/bootstrap.css">
+  </head>
+  <body>
+    <div class="container">
+    <h2>Exercise: DOM Detective</h2>
+    <h3 class="page-header">CASE #1: Can She Develop It?</h3>
+    <ul>
+      <li>Navigate to the<a href="http://www.girldevelopit.com">GirlDevelopIt</a> site,
```

Comment:
this might have been wrong before, but Girl Develop It should be three separate words